homeassistant/data_entry_flow.py

```diff
@@ -345,6 +346,7 @@ def async_show_form(
         data_schema: vol.Schema = None,
         errors: dict[str, str] | None = None,
         description_placeholders: dict[str, Any] | None = None,
+        last_step: bool = True,
```

Comment:
This seems weird to default to `True`

Reply:
I would expect it to be defaulted to None and typed as bool | None.

Reply:
What should be the difference between True and None then?

Reply:
`None` means not set. True/False means the config flow explicitly passed in True/False.

Reply:
Fixed.